pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -85,28 +90,6 @@
     </ProjectReference>
   </ItemGroup>
   
-  <ItemGroup Condition="'$(PackageTargetRuntime)' != ''">
-    <ArchitectureSpecificNativeFile Include="$(DotNetHostBinDir)/dotnet$(ApplicationFileExtension)" />
```

Comment:
I don't see the $(ApplicationFileExtension) property being used any more.  Can it be removed?